modules/monitoring-getting-information-about-alerts-silences-and-alerting-rules.adoc

```diff
@@ -10,87 +10,79 @@ The Alerting UI provides detailed information about alerts and their governing a
+. Go to the *Observe* -> *Alerting* -> *Silences* page.
 
 . Optional: Filter the silences by name using the *Search by name* field.
 
 . Optional: Filter silences by state by selecting filters in the *Filter* list. By default, *Active* and *Pending* filters are applied.
 
-. Optional: Sort the silences by clicking one or more of the *Name*, *Firing Alerts*, and *State* column headers.
+. Optional: Sort the silences by clicking one or more of the *Name*, *Firing alerts*, *State*, and *Creator* column headers.
+
+. Select the name of a silence to view its *Silence details* page. The page includes the following details:
 
-. Select the name of a silence to navigate to its *Silence Details* page. The page includes the following details:
-+
---
 * Alert specification
 * Start time
 * End time
 * Silence state
 * Number and list of firing alerts
---
 
 *To obtain information about alerting rules in the Administrator perspective*:
 
-. Navigate to the *Observe* -> *Alerting* -> *Alerting Rules* page.
+. Go to the *Observe* -> *Alerting* -> *Alerting rules* page.
 
 . Optional: Filter alerting rules by state, severity, and source by selecting filters in the *Filter* list.
 
-. Optional: Sort the alerting rules by clicking one or more of the *Name*, *Severity*, *Alert State*, and *Source* column headers.
+. Optional: Sort the alerting rules by clicking one or more of the *Name*, *Severity*, *Alert state*, and *Source* column headers.
+
+. Select the name of an alerting rule to view its *Alerting rule details* page. The page provides the following details about the alerting rule:
 
-. Select the name of an alerting rule to navigate to its *Alerting Rule Details* page. The page provides the following details about the alerting rule:
-+
---
-** Alerting rule name, severity, and description
-** The expression that defines the condition for firing the alert
-** The time for which the condition should be true for an alert to fire
-** A graph for each alert governed by the alerting rule, showing the value with which the alert is firing
-** A table of all alerts governed by the alerting rule
---
+* Alerting rule name, severity, and description
+* The expression that defines the condition for firing the alert
+* The time for which the condition should be true for an alert to fire
+* A graph for each alert governed by the alerting rule, showing the value with which the alert is firing
+* A table of all alerts governed by the alerting rule
 
 *To obtain information about alerts, silences, and alerting rules in the Developer perspective*:
 
-. Navigate to the *Observe* -> *<project_name>* -> *Alerts* page.
+. Go to the *Observe* -> *<project_name>* -> *Alerts* page.
 
 . View details for an alert, silence, or an alerting rule:
 
-* *Alert Details* can be viewed by selecting *>* to the left of an alert name and then selecting the alert in the list.
+* *Alert details* can be viewed by clicking *>* next to an alert name and then selecting the alert from the list.
```

Comment:
Accessiblity wise, screen readers might not call out this symbo, so maybe it's worth describing it in parentheses?

IBM SG: 

![Screenshot from 2024-03-06 12-25-47](https://github.com/openshift/openshift-docs/assets/57954076/ec420d19-d067-4324-ad75-f28e1712bcf1)

Reply:
Great idea, thanks for the tip!

Reply:
I found a great example here: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-symbols#individual-punctuation-marks-or-special-characters
so I will use that one.